dev/TabView/TabView.idl

```diff
@@ -7,6 +7,16 @@ enum TabViewWidthMode
 {
```

Comment:
Should this also have an auto value? @teaP

Reply:
It could!

Reply:
Would it be a breaking change to introduce Auto as the first value in this enum?

Reply:
There are to possible ways we could define  `Auto` here:
1. Let it be the default way for the control, i.e. it maps to an existing value
2. Let it behave like the `Auto` mode for NavigationView: if we have enough space for the TabViewItems, show them in SizeToContent, if we would need scrolling to show all of them, behave like compact

Reply:
Yes, it would be an API change, and I think any API change counts as a breaking change. I don't think we'd want to replace the current top value with Auto, we'd be adding Auto.

Reply:
@stmoy We can add Auto to the end of the list, but what is the expectation for Auto ? Can you clarify which behavior it would map to and when ? Also, perhaps we can do this as a follow up if it is not really about getting parity with WCT.

Reply:
The usual reason for the "Auto" value is "The default mode of the framework". In this case it should map to equal sizing if I remember correctly.

Reply:
Is there value in adding an "auto" if it is going to map to the default all the time ? I can see it being valuable if the behavior is going to be different in different scenarios and the developer does not want to deal with that. But they can do the same now but not setting this property. I suppose it gives us some leeway in the future if we want to introduce some change in behavior for auto.

Reply:
I like the idea of adding an auto, especially if it had some "smarts" baked in (like @chingucoding mentions with #2 above). However, I agree with @ranjeshj that we don't need an `auto` value to reach parity with WCT - so I think we should hold off on introducing it for now since there's no immediate need.